src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTestForFailedStatus.java

```diff
@@ -228,20 +230,18 @@ public void should_record_signature_failure_when_zip_contains_invalid_signature(
 
         // then
         envelopeWasNotCreated();
-        eventWasCreated(Event.DOC_SIGNATURE_FAILURE);
+        eventsWereCreated(ZIPFILE_PROCESSING_STARTED, DOC_SIGNATURE_FAILURE);
         errorWasSent(SAMPLE_ZIP_FILE_NAME, ErrorCode.ERR_SIG_VERIFY_FAILED);
     }
 
-    private void eventWasCreated(Event event) {
-        List<ProcessEvent> processEvents = processEventRepository.findAll();
-        assertThat(processEvents).hasSize(1);
-
-        ProcessEvent processEvent = processEvents.get(0);
-
-        assertThat(processEvent.getContainer()).isEqualTo(testContainer.getName());
-        assertThat(processEvent.getEvent()).isEqualTo(event);
-        assertThat(processEvent.getId()).isNotNull();
-        assertThat(processEvent.getReason()).isNotBlank();
+    private void eventsWereCreated(Event event1, Event event2) {
```

Comment:
could be more generic and accept arbitrary number of events but I guess we always have 2 so it's fine for now